### PR TITLE
[WFLY-10069] Remove all slf4j dependencies from WildFly

### PR DESCRIPTION
**Pull request by jmesnil**
With WFCORE-3703, all SLF4J dependencies are declared in WildFly Core

JIRA: https://issues.jboss.org/browse/WFLY-10069

### CODE REVIEW

**Comment by jmesnil** [comment]
This PR can merged after the component upgrade for wildfly-core that will incorporate https://github.com/wildfly/wildfly-core/pull/3182

File: feature-pack/src/main/resources/modules/system/layers/base/org/slf4j/ext/main/module.xml [hunk 1]
```diff
@@ -1,40 +0,0 @@
...
-  ~ You should have received a copy of the GNU Lesser General Public
-  ~ License along with this software; if not, write to the Free
-  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
-  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
-  -->
-
-<module xmlns="urn:jboss:module:1.5" name="org.slf4j.ext">
-
-    <properties>
-        <property name="jboss.api" value="deprecated"/>
```
Comment (rsvoboda) [inline feature-pack/src/main/resources/modules/system/layers/base/org/slf4j/ext/main/module.xml]: This was deprecated in EAP 7.1.0.
For EAP 6.4.0 and 7.0.0 it was public module.

This cannot be removed now

**Review by rsvoboda — CHANGES_REQUESTED** [review]

File: feature-pack/src/main/resources/modules/system/layers/base/org/slf4j/ext/main/module.xml (hunk 1 above)
Comment (rsvoboda) [reply feature-pack/src/main/resources/modules/system/layers/base/org/slf4j/ext/main/module.xml]: ah, it's move of slf4j stuff to wf-core, not total removal. Ignore my comment please.

https://github.com/wildfly/wildfly-core/pull/3182/files#diff-0267876148955d4bc36a55f5c9fae951R25

**Comment by jamezp** [comment]
@mkouba Does Weld still use `ch.qos.cal10n`?

**Comment by mkouba** [comment]
@jamezp No, it doesn't.. for a long time.

**Review by jamezp — APPROVED** [review]

**Comment by gaol** [comment]
Looks like the slf4j artifacts are back again at [component-matrix/pom.xml](https://github.com/wildfly/wildfly/blob/master/component-matrix/pom.xml#L219-L222) when fixing https://issues.jboss.org/browse/WFLY-9911

**Comment by kabir** [comment]
Retest this please

**Comment by kabir** [comment]
@jmesnil needs a rebase

**Comment by jmesnil** [comment]
retest this please

**Pull request merged**